ui/package.json

```diff
@@ -131,5 +131,8 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
+  },
+  "dependencies": {
```

Comment:
So I think the reason things are still happy is that we're still loading moment / ember-moment - they would need to be removed here. I think there's a few other helpers from `ember-moment` we use too. `now`, `is-before` and `is-after` and they all take an `interval` arg that recalculates the time after the given interval. We'd need those ported as well if we are going to move away from `moment`.